src/shadowbox/docker/Dockerfile

```diff
@@ -37,7 +37,7 @@ RUN ROOT_DIR=/ yarn do shadowbox/server/build
 # shadowbox image
 FROM node@sha256:12b2154fb459fa5f42c54771524609db041e7ef3465935d0ca82940d2d72669d
 
-# Versions can be found at https://github.com/Jigsaw-Code/outline-ss-server/releases
+# Keep in sync with version in third_party/outline-ss-server
 ARG SS_VERSION=1.1.4
```

Comment:
I don't think we're using this anymore, but maybe it would be useful to set as a label?

Reply:
Removed the ARG. I think we could restore it if we parse the version in `third_party/outline-ss-server/METADATA` in the build script.